extracts/v_extracts.gsheets_battleboard.sql

```diff
@@ -42,31 +116,59 @@ SELECT c.df_employee_number
       ,c.google_email
       ,c.original_hire_date
 
+      ,s.course_name
+      ,s.credittype
+
       ,ROUND(e.[PM1], 2) AS [PM1]
       ,ROUND(e.[PM2], 2) AS [PM2]
       ,ROUND(e.[PM3], 2) AS [PM3]
 
-      ,ROUND(p.[PM4], 2) AS [Last Year Final]
+      ,ROUND(p.[PM4], 2) AS last_year_final
 
       ,i.answer AS itr_response
 
-      ,CASE
-        WHEN c.primary_on_site_department = 'Elementary'
-         AND g.student_grade_level IS NOT NULL
-             THEN CONCAT(c.primary_on_site_department, ', Grade ', g.student_grade_level)
-        ELSE c.primary_on_site_department
-       END AS department_grade
+      ,CONCAT(staffing_model_id,'-',seat_number) AS staffing_model_id
+      
 FROM gabby.people.staff_crosswalk_static c
-LEFT JOIN etr_pivot e
+LEFT JOIN etr_pivot e 
   ON c.df_employee_number = e.df_employee_number
  AND e.academic_year = gabby.utilities.GLOBAL_ACADEMIC_YEAR()
 LEFT JOIN etr_pivot p
   ON c.df_employee_number = p.df_employee_number
  AND p.academic_year = gabby.utilities.GLOBAL_ACADEMIC_YEAR() - 1
-LEFT JOIN elementary_grade g 
-  ON c.df_employee_number = g.employee_number
+LEFT JOIN seat_number s
+  ON c.df_employee_number = s.df_employee_number
 LEFT JOIN gabby.surveys.intent_to_return_survey_detail i
   ON c.df_employee_number = i.respondent_df_employee_number
  AND i.question_shortname = 'intent_to_return'
  AND i.campaign_academic_year = gabby.utilities.GLOBAL_ACADEMIC_YEAR()
-WHERE c.[status] IN ('Active','Leave','Prestart')
+WHERE c.[status] IN ('Active','Leave','Prestart') 
+ AND c.legal_entity_name <> 'KIPP TEAM and Family Schools Inc.'
+ AND c.primary_site NOT IN ('Room 9 - 60 Park Pl','Room 10 - 121 Market St','Room 11 - 1951 NW 7th Ave')
+ AND c.primary_site NOT LIKE '%Campus%'
+ )
+
+SELECT m.include AS seat_open
+      ,m.academic_year
+      ,m.staffing_model_id
+      ,m.display_name
+      
+      ,r.staffing_model_id AS generated_id
+      ,r.course_name AS powerschool_course
+      ,r.credittype AS powerschool_credit
+      ,r.df_employee_number
+      ,r.preferred_name
+      ,r.primary_site
+      ,r.primary_job
+      ,r.google_email
+      ,r.original_hire_date
+      ,r.PM1
+      ,r.PM2
+      ,r.PM3
+      ,r.last_year_final
+      ,r.itr_response
+      ,r.[status]
+
+FROM gabby.people.staffing_model m
+FULL OUTER JOIN current_roster r
```

Comment:
avoid FULL OUTER JOIN, it can be unpredictable. what's the purpose of joining it this way?